test/tests/dom/getCurrentScriptUID.js

```diff
@@ -26,15 +26,15 @@ describe('get current script UID', () => {
             );
         }
     });
-  
+
     it('should use script\'s src and attributes to create the script UID', () => {
 
         const currentScript : HTMLScriptElement = getCurrentScript();
         const { src, dataset } = currentScript;
         const stringToHash = JSON.stringify({ src, dataset });
-        const hashedString = hashStr(stringToHash);
+        const hashedString = strHashStr(stringToHash).slice(20);
```

Comment:
Non-blocking suggestion, maybe worth it to store such value in a constant (avoiding 'maigc numbers')..
what about:

```
const MAX_STRING_LENGHT = 20;
```